web/ui/src/pages/Clustering.tsx

```diff
@@ -0,0 +1,17 @@
+import { faCubes } from '@fortawesome/free-solid-svg-icons';
+
+import PeerList from '../features/clustering/PeerList';
+import Page from '../features/layout/Page';
+import { usePeerInfo } from '../hooks/peerInfo';
+
+function PageClusteringPeers() {
+  const peers = usePeerInfo();
+
+  return (
+    <Page name="Clustering" desc="List of clustering peers." icon={faCubes}>
```

Comment:
nit: Don't put a period on an h2:

```suggestion
    <Page name="Clustering" desc="List of clustering peers" icon={faCubes}>
```